packages/api/src/record.ts

```diff
@@ -77,16 +94,45 @@ export class Record implements RecordModel {
   // Private variables for DWN `RecordsWrite` message properties.
   private _author: string;
   private _attestation?: RecordsWriteMessage['attestation'];
+  private _authorization?: RecordsWriteMessage['authorization'];
   private _contextId?: string;
   private _descriptor: RecordsWriteDescriptor;
   private _encryption?: RecordsWriteMessage['encryption'];
+  private _initialWrite: RecordOptions['initialWrite'];
+  private _initialWriteStored: boolean;
   private _recordId: string;
-
+  private _protocolRole: RecordOptions['protocolRole'];
   // Getters for immutable DWN Record properties.
 
+  static sendCache = new Map();
+  static sendCacheLimit = 100;
+  static setSendCache(recordId, target){
```

Comment:
Please add some documentation that explains what this does.
```suggestion
  static setSendCache(recordId, target) {
```

Reply:
Added